package.json

```diff
@@ -8,18 +8,30 @@
     "url": "http://github.com/CSNW/movie-app"
   },
   "scripts": {
-    "test": "mocha test.js"
+    "preinstall": "npx npm-force-resolutions",
+    "postinstall": "cd client && npm install",
+    "client": "npm start --prefix client",
+    "dev": "concurrently \"nodemon --inspect=5858 server.js\" \"npm run client\"",
```

Comment:
@speck14 It'd be important to note in the README that you are relying on nodemon being installed globally.  Just a small block about installing it like `npm install -g nodemon` if you don't already have it.